docs/blog/author.yaml

```diff
@@ -105,3 +105,11 @@
   bio: Full-stack developer, clean code enthusiast, blogger at https://www.vojtechruzicka.com/
   avatar: avatars/vojtech-ruzicka.jpg
   twitter: "@vojtechruzicka"
+- id: Robert Svensson
+  bio: Generally winning
```

Comment:
Could you make your bios a bit more descriptive? ;-)

And mention that you work at Contentful.

Reply:
Yes, sorry! I meant to change that, and completely forgot.